tdrs-backend/tdpservice/parsers/test/factories.py

```diff
@@ -25,3 +28,168 @@ class Meta:
 
     object_id = 1
     content_type_id = 1
+
+class TanfT1Factory(factory.django.DjangoModelFactory):
+    """Generate TANF T1 record for testing."""
+
+    class Meta:
+        """Hardcoded meta data for TANF_T1."""
+
+        model = "search_indexes.TANF_T1"
+
+    RecordType = fake.uuid4()
+    RPT_MONTH_YEAR = 1
+    CASE_NUMBER = 1
+    COUNTY_FIPS_CODE = 1
+    STRATUM = 1
+    ZIP_CODE = 1
+    FUNDING_STREAM = 1
+    DISPOSITION = 1
+    NEW_APPLICANT = 1
+    NBR_FAMILY_MEMBERS = 1
+    FAMILY_TYPE = 1
+    RECEIVES_SUB_HOUSING = 1
+    RECEIVES_MED_ASSISTANCE = 1
+    RECEIVES_FOOD_STAMPS = 1
+    AMT_FOOD_STAMP_ASSISTANCE = 1
+    RECEIVES_SUB_CC = 1
+    AMT_SUB_CC = 1
+    CHILD_SUPPORT_AMT = 1
+    FAMILY_CASH_RESOURCES = 1
+    CASH_AMOUNT = 1
+    NBR_MONTHS = 1
+    CC_AMOUNT = 1
+    CHILDREN_COVERED = 1
+    CC_NBR_MONTHS = 1
+    TRANSP_AMOUNT = 1
+    TRANSP_NBR_MONTHS = 1
+    TRANSITION_SERVICES_AMOUNT = 1
+    TRANSITION_NBR_MONTHS = 1
+    OTHER_AMOUNT = 1
+    OTHER_NBR_MONTHS = 1
+    SANC_REDUCTION_AMT = 1
+    WORK_REQ_SANCTION = 1
+    FAMILY_SANC_ADULT = 1
+    SANC_TEEN_PARENT = 1
+    NON_COOPERATION_CSE = 1
+    FAILURE_TO_COMPLY = 1
+    OTHER_SANCTION = 1
+    RECOUPMENT_PRIOR_OVRPMT = 1
+    OTHER_TOTAL_REDUCTIONS = 1
+    FAMILY_CAP = 1
+    REDUCTIONS_ON_RECEIPTS = 1
+    OTHER_NON_SANCTION = 1
+    WAIVER_EVAL_CONTROL_GRPS = 1
+    FAMILY_EXEMPT_TIME_LIMITS = 1
+    FAMILY_NEW_CHILD = 1
+
+
+class TanfT2Factory(factory.django.DjangoModelFactory):
+    """Generate TANF T2 record for testing."""
+
+    class Meta:
+        """Hardcoded meta data for TANF_T2."""
+
+        model = "search_indexes.TANF_T2"
+
+    RecordType = fake.uuid4()
+    RPT_MONTH_YEAR = 1
+    CASE_NUMBER = '1'
+    FAMILY_AFFILIATION = 1
+    NONCUSTODIAL_PARENT = 1
+    DATE_OF_BIRTH = 1
+    SSN = '1'
+    RACE_HISPANIC = "1"
+    RACE_AMER_INDIAN = "1"
+    RACE_ASIAN = "1"
+    RACE_BLACK = "1"
+    RACE_HAWAIIAN = "1"
+    RACE_WHITE = "1"
+    GENDER = 1
+    FED_OASDI_PROGRAM = 1
+    FED_DISABILITY_STATUS = 1
+    DISABLED_TITLE_XIVAPDT = 1
+    AID_AGED_BLIND = 1
+    RECEIVE_SSI = 1
+    MARITAL_STATUS = "1"
+    RELATIONSHIP_HOH = 1
+    PARENT_WITH_MINOR_CHILD = "1"
+    NEEDS_PREGNANT_WOMAN = 1
+    EDUCATION_LEVEL = "01"
+    CITIZENSHIP_STATUS = "1"
+    COOPERATION_CHILD_SUPPORT = "1"
+    MONTHS_FED_TIME_LIMIT = 2
+    MONTHS_STATE_TIME_LIMIT = 2
+    CURRENT_MONTH_STATE_EXEMPT = 1
+    EMPLOYMENT_STATUS = "1"
+    WORK_ELIGIBLE_INDICATOR = "01"
+    WORK_PART_STATUS = "1"
```

Comment:
item # 49 (work_part_status) is 2 digits. takes on the following possible values: `01,02,05,07,09,15,15,17,18,19,99`